.storybook/main.ts

```diff
@@ -0,0 +1,34 @@
+module.exports = {
```

Comment:
Why do we need this change? Is it because of the later `@babel/core` version or something else?